packages/react-animation/src/useExposeAnimation/useExposeAnimation.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from 'react';
+
+const animations = new Map<unknown, gsap.core.Animation>();
```

Comment:
This "global" scope is not really global, but scoped to the module. It could happen (e.g. with multiple chunks) that this file is compiled twice, and that different instances have their own version of this store.

Best use the `global` to store it on the node/window context.

Reply:
Updated, used `globalThis` as that namespace should be available in all environments.